reporter/otlp_reporter.go

```diff
@@ -738,19 +743,26 @@ func createFunctionEntry(funcMap map[funcInfo]uint64,
 }
 
 // getSampleAttributes builds a sample-specific list of attributes.
-func getSampleAttributes(profile *profiles.Profile, k traceAndMetaKey) []uint64 {
+func getSampleAttributes(profile *profiles.Profile,
+	k traceAndMetaKey, attributeMap map[string]uint64) []uint64 {
```

Comment:
Minor nit to avoid confusion with the other `k`

```suggestion
	traceKey traceAndMetaKey, attributeMap map[string]uint64) []uint64 {
```